shared/src/main/scala/pl/metastack/metatime/Date.scala

```diff
@@ -75,17 +68,120 @@ trait Date extends Year with Month with Day {
     Year(y).unix().value +
     Month(m).unix().value +
     Day(d).unix().value)
-  }
+
+}
 
 object Date {
-  def now(): Date = ???
-  def apply(year: Int): Date = ???
+  def apply(year: Int): Date = new Date {
+    override val y: Int = year
+    override val m: Int = 1
+    override val d: Double = 1.0
+  }
   def apply(year: Int, month: Int): Date = ???
   def parse(format: String, timezone: Timezone): Date = ???
 
-  def apply(year: Int = 1970,
+  def findYearsOrRemainingDaysOfYear(totalDays: Int, year: Boolean) : Int = {
```

Comment:
This function can be removed altogether because the developer can just call `yearsPassed` or `daysPassed` right away.